openapi.yaml

```diff
@@ -0,0 +1,1667 @@
+openapi: 3.0.3
```

Comment:
Maybe replace this file with a symlink into kopfsachen-dev. Make sure to document where this repo has to be checked out to